avro/src/main/java/com/fasterxml/jackson/dataformat/avro/AvroAnnotationIntrospector.java

```diff
@@ -63,6 +65,15 @@ public PropertyName findNameForDeserialization(Annotated a) {
     }
 
     @Override
+    public Object findDeserializer(Annotated am) {
+        AvroEncode ann = _findAnnotation(am, AvroEncode.class);
+        if (ann != null) {
+            return CustomEncodingDeserializer.class;
```

Comment:
Hmmh. What do you mean by not supporting that? Since its `Object` it should, and if not, that should be fixed...